.gitignore

```diff
@@ -26,5 +35,27 @@ visualization.*/addon.xml
 # clion
 .idea/
 
-# KDev
-*.kdev4
\ No newline at end of file
+*.o
+*.a
+*.P
+*.lo
+*.la
+*.pvr
+*.zip
+*.patch
```

Comment:
do not ignore patch files
```suggestion
```

Reply:
Also some others, has c&p from another and not seen direct, was mainly updated to prevent the debian files.

What you think about `/*.patch` (like on kodi itself), to prevent the bat add after a `git format-patch ...`?

Reply:
I wouldn't add that much to the gitignore. Just the generated files in debian folder and a build dir (which is already present). No need for .o, .a, .zip, .exe, .lib, .dll, ... as it is recommended to build in a separate folder.

Reply:
Have cleaned up and only added to the debian files, a visual studio part and the `/*.patch`